FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.h

```diff
@@ -38,6 +38,7 @@ typedef NS_ENUM(NSInteger, FPRApplicationState) {
 /** This class is used to track the app activity and create internal traces to capture the
  *  performance metrics.
  */
+NS_EXTENSION_UNAVAILABLE_IOS("Firebase Performance is not supported for iOS extensions.")
```

Comment:
Should be `NS_EXTENSION_UNAVAILABLE("Firebase Performance is not supported for extensions.")`

I suspect the current version will fail to build for tvOS on Xcode 13.

Reply:
I tested for tvOS and the build succeeded, but I agree that `NS_EXTENSION_UNAVAILABLE` should be the right one.

Reply:
I might be confusing macOS and tvOS issues I saw before. Anyway, agreed `NS_EXTENSION_UNAVAILABLE` is better and will make adding macOS easier as well as if tvOS adds extensions

Reply:
I honestly did not like this MACRO being spread all over the code. We need to find a better elegant solution to this.

Can we huddle for a better solution here? I don't want to block this PR. But this can't scale.

Reply:
I could not come up with a dramatically different alternative. As slight maintainability improvement you may declare a an alias macro like e.g. [here](https://github.com/firebase/firebase-ios-sdk/blob/ec193d015078cfdf084aa9714da6e8838ac2058b/FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheckAvailability.h#L41) to simplify potential changes in the future (you will have to change availability only at one place instead of updating all the affected files.